Gemfile-rails-dependencies

```diff
@@ -16,7 +16,7 @@ if RUBY_VERSION.to_f < 3
   # sqlite3 1.7.x doesn't work on all platforms for Ruby 2.x
   gem 'sqlite3', '~> 1.4', '< 1.7', platforms: [:ruby]
 else
-  gem 'sqlite3', '~> 1.4', platforms: [:ruby]
+  gem 'sqlite3', '>= 1.4', platforms: [:ruby]
```

Comment:
What if we required 2.0 only on Rails 8, which [is](https://github.com/rspec/rspec-rails/blob/d0e322b20bb713b200b66d4a7cc21a272d0b4374/.github/workflows/ci.yml#L39) `main` and did that in a `case` below on line 28?
The other file would need similar changes

Reply:
Hi @pirj .
Should I just close my PR's in favor of: https://github.com/rspec/rspec-rails/pull/2784 ?

Reply:
That other one is not merged yet, I’d keep this one open. Do you want to push this one forward with findings from #2784?